log-viewer/modules/TreeView.ts

```diff
@@ -310,16 +317,19 @@ function renderTree() {
   }
 }
 
-function goToFile(evt: Event) {
+function goToFile(evt: MouseEvent) {
   const elem = evt.target as HTMLElement;
   const target = elem.matches("a") ? elem.parentElement?.parentElement : null;
+  showBreadcrumb(elem);
   const timeStamp = target?.dataset.enterstamp;
   if (timeStamp) {
-    const node = findByTimeStamp(treeRoot, timeStamp);
-    if (node) {
-      const fileOpenInfo = deriveOpenInfo(node);
-      if (fileOpenInfo) {
-        hostService().openType(fileOpenInfo);
+    if (evt.metaKey && !evt.altKey && !evt.shiftKey) {
```

Comment:
I think for it to work on ctrl + click on windows we need this. Although I think this would mean windows key or cmd key and click works. and ctrl click works on both.
```suggestion
    if (evt.ctrlKey || evt.metaKey) {
```

Reply:
This doesn't look equivalent. Do we want to ignore alt and shift modifiers?

Reply:
Also, ctrl-click seems to open the right-click menu...

Reply:
Changed to:
```suggestion
    if ((evt.ctrlKey || evt.metaKey) && !evt.altKey && !evt.shiftKey) {
```